.github/workflows/github-actions-pull-request.yml

```diff
@@ -3,6 +3,8 @@ on: pull_request
 jobs:
   execute-tests:
     runs-on: ubuntu-latest
+    env:
```

Comment:
Are you sure you want to add this?

Reply:
Yes, for machines where NEAL is not a package, a fresh install of all its dependencies is needed. This env var increases the timeout, which is 60 seconds by default.